src/app/documentation/demos/wizard/wizard-design.demo.ts

```diff
@@ -7,6 +7,15 @@
 import { Component, ViewChild } from "@angular/core";
 import { Wizard, WizardPage } from "clarity-angular";
 
+import { FocusTrapTracker } from "clarity-angular/utils/focus-trap/focus-trap-tracker.service";
+
+class FocusTrapServiceInterjectory {
```

Comment:
Does it make sense to make `FocusTrapServiceInterjectory` a class we can import? Seems like we are already duplicating the code...

Reply:
Indeed, I will address these issues you've pointed out.